esp/platform/espcontext.cpp

```diff
@@ -820,6 +820,22 @@ static IEspContainer*& getContainer()
     return gContainer;
 }
 
+LogRequest readLogRequest(char const* req)
+{
+    if (isEmptyString(req))
+        return LogRequestsNever;
+
+    if (strieq(req, "all"))
+        return LogRequestsAlways;
+    if (strieq(req, "never"))
+        return LogRequestsNever;
+    if (strieq(req, "only-ones-with-issues"))
+        return LogRequestsWithIssuesOnly;
+    if (strieq(req, "true"))
```

Comment:
should probably support "1" as a boolean true value.

Reply:
There's a standard function we use for such situations (I can't remember its name)

Reply:
I guess I can use getEnum().